src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+~NuGet.PackageManagement.NuGetPackageManager.PreviewProjectsInstallPackageAsync(System.Collections.Generic.IReadOnlyCollection<NuGet.ProjectManagement.NuGetProject> nuGetProjects, NuGet.Packaging.Core.PackageIdentity packageIdentity, NuGet.PackageManagement.ResolutionContext resolutionContext, NuGet.ProjectManagement.INuGetProjectContext nuGetProjectContext, System.Collections.Generic.IReadOnlyCollection<NuGet.Protocol.Core.Types.SourceRepository> activeSources, NuGet.Versioning.VersionRange versionRange, System.Threading.CancellationToken token) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<NuGet.PackageManagement.ResolvedAction>>
```

Comment:
This is the same API as was removed from the shipping txt file, so should be undeleted from there, rather than added here.